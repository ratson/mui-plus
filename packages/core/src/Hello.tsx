import React from 'react'
import { Box } from '@material-ui/core'

export type HelloProps = {}

const Hello: React.FC<HelloProps> = ({ children }) => {
  return <Box>Hello: {children}</Box>
}

export default Hello
