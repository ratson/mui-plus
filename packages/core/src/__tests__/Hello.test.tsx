import React from 'react'
import renderer from 'react-test-renderer'
import Hello from '../Hello'

describe('Hello', () => {
  test('Say Hello', () => {
    const component = renderer.create(<Hello />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
