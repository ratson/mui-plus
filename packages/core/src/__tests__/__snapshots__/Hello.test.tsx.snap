// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello Say Hello 1`] = `
<div
  className="MuiBox-root MuiBox-root-2"
>
  Hello: 
</div>
`;
